### PR TITLE
Update some words already translated

### PR DESCRIPTION
**Pull request by thomasbnt**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

The **French locale** are some wrong translated words. 
## Related Tickets & Documents
Yes, this DEV post

https://dev.to/devteam/forem-hacktoberfest-let-s-internationalize-404n

image

## QA Instructions, Screenshots, Recordings

Little change like **Créer Publication** to **Créer une publication**. 
**Créer un nouveau post** is also good at french. We like english words here 😄

image

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: Just translated some words, no touched other files.
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: config/locales/fr.yml [hunk 1]
```diff
@@ -18,22 +18,22 @@ fr:
...
     like: Aime
     listings: Annonces
-    log_in: Entrer
+    log_in: Se connecter
     month: Mois
     more_from: Plus de
-    my_tags: Mes Sujets
-    popular_tags: Sujets Populaire
+    my_tags: Mes sujets
+    popular_tags: Sujets populaire
```
Comment (aurelieverrot) [inline config/locales/fr.yml:25]:
```suggestion
    popular_tags: Sujets populaires
```
;-)

File: config/locales/fr.yml [hunk 2]
```diff
@@ -18,22 +18,22 @@ fr:
     latest: Récent
     like: Aime
     listings: Annonces
-    log_in: Entrer
+    log_in: Se connecter
     month: Mois
     more_from: Plus de
```
Comment (aurelieverrot) [inline config/locales/fr.yml:23]: You might want to change this one as well.
It is used in this context: 
image
Example in [page](https://dev.to/devteam/hacktoberfest-2021-is-here-4a3l).

Using "Plus de The DEV Team" would mean "more The DEV Team".
In this context, I would use "Plus de la part de The DEV Team".

**Review by aurelieverrot — COMMENTED** [review]
Nice translations 💯 
I thought I could help a little with this PR 😊

**Comment by msarit** [comment]
@nickytonline @ellativity Hey Hey! Could we have your review of this French-related PR? 🙏🏾

**Review by nickytonline — APPROVED** [review]
Merci pour les modifs @thomasbnt! 🚢

**Comment by thomasbnt** [comment]
> Merci pour les modifs @thomasbnt! 🚢

I can always translate for Forem, it makes me happy 💪🏼💚

**Review by rhymes — APPROVED** [review]
Merci @thomasbnt :)

**Comment by rhymes** [comment]
@thomasbnt could you merge `main` in your PR? It contains a fix required to unblock the failing build. Thanks :(

**Comment by thomasbnt** [comment]
> @thomasbnt could you merge `main` in your PR? It contains a fix required to unblock the failing build. Thanks :(

No problem 😄

**Pull request merged**